llvm/test/CodeGen/CHERI-Generic/Inputs/landingpad-non-preemptible.ll

```diff
@@ -0,0 +1,130 @@
+; RUN: llc @PURECAP_HARDFLOAT_ARGS@ < %s -o - | FileCheck %s
+; RUN: llc @PURECAP_HARDFLOAT_ARGS@ < %s -o - -filetype=obj | llvm-readobj -r -t - | FileCheck %s --check-prefix=RELOCS
```

Comment:
Is hard float what we use in general even if it doesn't make a difference?

Reply:
I've been using that as the default lately, but can also change it. Or I could add a @DEFAULT_PURECAP_ARGS@ substitution?

Reply:
It's fine if that's what we're using, just wanted to check.